.env.template

```diff
@@ -66,6 +66,11 @@ CDR_PAYMENT_REDIRECTION_URL: str | None
 MYECLPAY_MAXIMUM_WALLET_BALANCE: int | None
 #MYECLPAY_DATA_VERIFIER_ACCESS_TOKEN = ""
 
+# S3 configuration for file storage
+S3_BUCKET_NAME = "hyperion"
+S3_ACCESS_KEY_ID = "realaccesskey"
+S3_SECRET_ACCESS_KEY = "realsecretkey"
```

Comment:
```suggestion
#S3_BUCKET_NAME = "hyperion"
#S3_ACCESS_KEY_ID = "realaccesskey"
#S3_SECRET_ACCESS_KEY = "realsecretkey"
```

Reply:
Can the bucket be used for something else than MyECLPay logs? If no, we should use a more explicit name like `MYECLPAY_LOGS_S3_BUCKET_NAME`